lib/omniauth/strategies/saml.rb

```diff
@@ -10,7 +10,7 @@ def self.inherited(subclass)
         OmniAuth::Strategy.included(subclass)
       end
 
-      OTHER_REQUEST_OPTIONS = [:skip_conditions, :allowed_clock_drift, :matches_request_id, :skip_subject_confirmation].freeze
+      OTHER_REQUEST_OPTIONS = [:skip_conditions, :allowed_clock_drift, :matches_request_id, :skip_subject_confirmation, :soft].freeze
```

Comment:
Is this line necessary? Looking at the `ruby-saml` source, it seems that `soft` gets picked up from `settings`.